cron.py

```diff
@@ -757,6 +761,72 @@ def delete_old_monitoring():
     LOG.d("delete monitoring records older than %s, nb row %s", max_time, nb_row)
 
 
+def check_hibp():
+    """
+    Check all aliases on the HIBP (Have I Been Pwned) API
+    """
+    LOG.d("Checking HIBP API for aliases in breaches")
+
+    if len(HIBP_API_KEYS) == 0:
+        LOG.exception("No HIBP API keys")
+        return
+
+    api_key_index = 0
+
+    LOG.d("Updating list of known breaches")
+    r = requests.get("https://haveibeenpwned.com/api/v3/breaches")
+    for entry in r.json():
+        hibp_entry = Hibp.get_by(name=entry['Name'])
+        if hibp_entry:
+            hibp_entry.description = entry['Description']
+        else:
+            hibp_entry = db.session.add(
+                Hibp(
+                    name=entry['Name'],
+                    description=entry['Description']
+                )
+            )
+
+    db.session.commit()
+
+    for alias in Alias.query.all():
+        LOG.d("Checking %s on HIBP", alias.email)
```

Comment:
Nits: we can just use `alias` instead of `alias.email` as this usually gives more info and shorter to type :)